generator/uscore/uscore_generator.rb

```diff
@@ -158,8 +158,15 @@ def create_search_test(sequence, search_param)
 
               skip 'No resources appear to be available for this patient. Please use patients with more information.' unless @resources_found
 
-              @#{sequence[:resource].downcase} = reply.try(:resource).try(:entry).try(:first).try(:resource)
+              @#{sequence[:resource].downcase} = reply&.resource&.entry&.first&.resource
               @#{sequence[:resource].downcase}_ary = reply&.resource&.entry&.map { |entry| entry&.resource }
+              page_count = 1
```

Comment:
I think this whole thing should be a method (unfortunately it would live in `SequenceBase`, so that this code isn't duplicated in each sequence. Something like:
```
@#{sequence[:resource].downcase}_ary = fetch_all_search_results(reply&.resource)
```
That would also allow this method to be unit tested, which I think we need since we've discussed error handling around this functionality as something of particular importance.

Reply:
I moved it to SequenceBase, but I'm not sure what the tests would look like for it. Should I try something where next_bundle never returns or maybe returns an error? Not really sure how to set that up.

Reply:
Let's get together in person and go over it.